src/librustc/metadata/creader.rs

```diff
@@ -49,15 +49,16 @@ impl<'a, 'v> visit::Visitor<'v> for CrateReader<'a> {
 }
 
 fn dump_crates(cstore: &CStore) {
-    debug!("resolved crates:");
```

Comment:
Was the switch from `debug!` to `info!` (and likewise from `log::DEBUG` to `log::INFO`) deliberate, or an accident?

I would have thought we could (and should) leave it as it was, under `debug!`